build/cd-build.yml

```diff
@@ -0,0 +1,39 @@
+# Starter pipeline
```

Comment:
As discussed, I would leave it out if it doesn't work yet and do a dedicated PR

Reply:
removed it